json-rpc/src/methods.rs

```diff
@@ -534,7 +536,7 @@ async fn get_account_transactions(
 
         all_txs.push(TransactionView {
             version: tx.version,
-            hash: tx.transaction.hash().to_hex(),
+            hash: BytesView::from(tx.transaction.hash().to_vec()),
             bytes: lcs::to_bytes(&tx.transaction)?.into(),
```

Comment:
see here,  if we declare everything in response field as bytesview directly we could sae all these BytesView::from()